bin/index.js

```diff
@@ -45,6 +45,16 @@ config(function (versions) {
   }
 
   async.doWhilst(getBin, whileCondition, function () {
-    if (switcherBin) return switcher(switcherBin)
+    if (switcherBin) {
+      var switcherProcess = switcher(switcherBin)
+
+      switcherProcess.on('close', function (code) {
+        if (code === 0) {
+          console.log(maxSatisfyVersion)
```

Comment:
To have the version echoed to the stdout and then beeing able to get it in a bash variable